test/api_test.lua

```diff
@@ -49,3 +50,21 @@ g.test_role_statistics = function()
     local result = g.queue_conn:eval(cmd)
     t.assert_type(result, 'table')
 end
+
+g.test_role_statistics_read_only_router = function()
```

Comment:
I suggest to add an assert that such instance is actually read_only.

Reply:
done, its a good idea, thank you